ykbuild/build.rs

```diff
@@ -18,6 +18,9 @@ fn main() {
         return;
     }
 
+    if !metadata("YKLLVM_SUBMODULE_PATH").is_ok() {
```

Comment:
This uses `YKLLVM_SUBMODULE_PATH` as a literal, so surely always fails? I think the idiomatic version would be `if Path::new(YKLLVM_SUBMODULE_PATH).is_dir()`.

Reply:
done :point_right: https://github.com/ykjit/yk/pull/748/commits/11b5f01ac4087b7559dfee27a5f61c0518ddc860